moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -20,16 +25,24 @@ export const useGetProjectListView = ({ columnsToReturn, queryConfig }) => {
     defaultOrderByDirection: queryConfig.order.defaultOrderByDirection,
   });
 
+  const { searchTerm, setSearchTerm, searchWhereString } = useSearch({
+    queryConfig,
+    defaultSearchTerm,
+  });
+
```

Comment:
After working more on the advanced search, I'm starting to feel like the search, pagination, etc custom hooks should be called at the top level of `ProjectListViewTable` so that `useProjectListView` is only responsible for receiving the return values to assemble a query and returning a query that Apollo `useQuery` can then use. I'm going to test it out in the next PR and see where it goes, but it would look something like this.

```js
const { searchTerm, setSearchTerm, searchWhereString } = useSearch({
    queryConfig,
    defaultSearchTerm,
  });

const { query } = useProjectListView({
  queryLimit,
  queryOffset,
  orderByColumn,
  orderByDirection,
  searchWhereString,
});

```

Reply:
💯 i like it a lot!